recipe/meta.yaml

```diff
@@ -1,46 +1,49 @@
-{% set version = "20160523b" %}
+{% set version = "20170520" %}
 
 package:
   name: texlive-core
   version: {{ version }}
 
 source:
   fn: texlive-{{ version }}-source.tar.xz
-  url: http://mirrors.ctan.org/systems/texlive/Source/texlive-{{ version }}-source.tar.xz
-  sha256: a8b32ca47f0a403661a09e202f4567a995beb718c18d8f81ca6d76daa1da21ed
+  url: http://mirrors.concertpass.com/tex-archive/systems/texlive/Source/texlive-{{ version }}-source.tar.xz
```

Comment:
Looking e.g. at https://src.fedoraproject.org/cgit/rpms/texlive.git/tree/texlive.spec gives me shivers. What could probably be done without too much work is create a `texlive` metapackage with 2 subpackages: `texlive-core` (containing more or less the same of this one) and `texlive-extras`.